src/components/interaction-designer/resource-viewer/ResourceViewerBlock.vue

```diff
@@ -34,18 +44,21 @@ import {PropType} from 'vue'
 
 export default {
   name: 'ResourceViewerBlock',
+  mixins: [lang],
   props: {
-    block: Object as PropType<IBlock>,
+    block: {
+      required: true,
+      type: Object as PropType<IBlock>,
+    },
   },
-  mixins: [lang],
   computed: {
     ...mapGetters('flow', ['block_classesConfig']),
     ...mapGetters('builder', ['isEditable']),
     blockLabel() {
       return this.block.label > '' ? this.block.label : lang.trans('flow-builder.untitled-block')
     },
     hasContent() {
-      return this.block_classesConfig[this.block.type].is_interactive === true
+      return this.block.config.prompt !== undefined
```

Comment:
main change

Reply:
@object-required 
You used another criteria in voto5 for this:
```
...
    const promptValidationStatus = rootState.validation.validationStatuses[`resource/${block?.config?.prompt}`]?.isValid

    return {
      isContentBlock: promptValidationStatus !== undefined,
      ...
    }
...
```

What would you suggest we could do to make them consistent? Keep two different ones / use my code / use your code?

Reply:
I believe `this.block.config.prompt !== undefined` is more straightforward, and it doesn't make us rely on the properly working validation. Let's go with yours :-)